core/store/src/main/java/uk/gov/gchq/gaffer/store/Store.java

```diff
@@ -1101,6 +1108,25 @@ private void addConfiguredOperationHandlers() {
     }
 
     protected void startCacheServiceLoader(final StoreProperties properties) {
-        CacheServiceLoader.initialise(properties.getProperties());
+        final String jobTrackerCacheClass = properties.getJobTrackerCacheServiceClass();
+        final String namedViewCacheClass = properties.getNamedViewCacheServiceClass();
+        final String namedOperationCacheClass = properties.getNamedOperationCacheServiceClass();
+        final String defaultCacheClass = properties.getDefaultCacheServiceClass();
+        final boolean defaultServiceRequired = !((jobTrackerCacheClass != null) && (namedViewCacheClass != null) && (namedOperationCacheClass != null));
+
+        if (jobTrackerCacheClass != null) {
+            CacheServiceLoader.initialise(JOB_TRACKER_CACHE_SERVICE_NAME, jobTrackerCacheClass, properties.getProperties());
+        }
+        if (namedViewCacheClass != null) {
+            CacheServiceLoader.initialise(NAMED_VIEW_CACHE_SERVICE_NAME, jobTrackerCacheClass, properties.getProperties());
```

Comment:
```suggestion
            CacheServiceLoader.initialise(NAMED_VIEW_CACHE_SERVICE_NAME, jobTrackerCacheClass, properties.getProperties());
```

Not sure this is meant to be 'jobTrackerCacheClass'? If it should be namedViewCacheClass, then the issue is repeated in the if statement after this as well.

Reply:
Good catch, now fixed